service/service_construction.go

```diff
@@ -194,8 +192,15 @@ func (s ConstructionService) ConstructionCombine(
 		return nil, wrapError(errInternalError, err)
 	}
 
+	wrappedSignedTx := signedTransactionWrapper{SignedTransaction: string(signedTxJSON), Currency: unsignedTx.Currency}
```

Comment:
Only possible nit could be that we are doing double marshaling here but 🤷 .

There is a small compatibility issue with a change like this where any inflight txs they have will no longer be "broadcastable" on this version.

Reply:
Something we'll want to call out

Reply:
Yeah, figured it was easier to just double marshal than worry about any of the eth transaction type.  Happy to change if you feel strongly

Reply:
Nah this is fine.